NuKeeper.Update/Process/UpdateDirectoryBuildTargetsCommand.cs

```diff
@@ -0,0 +1,64 @@
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using NuGet.Configuration;
+using NuGet.Versioning;
+using NuKeeper.Abstractions.Logging;
+using NuKeeper.Abstractions.NuGet;
+using NuKeeper.Inspection.RepositoryInspection;
+
+namespace NuKeeper.Update.Process
+{
+    public class UpdateDirectoryBuildTargetsCommand : IUpdateDirectoryBuildTargetsCommand
+    {
+        private readonly INuKeeperLogger _logger;
+
+        public UpdateDirectoryBuildTargetsCommand(INuKeeperLogger logger)
+        {
+            _logger = logger;
+        }
+
+        public Task Invoke(PackageInProject currentPackage, NuGetVersion newVersion, PackageSource packageSource,
+            NuGetSources allSources)
+        {
+            XDocument xml;
+            using (var xmlInput = File.OpenRead(currentPackage.Path.FullName))
+            {
+                xml = XDocument.Load(xmlInput);
+            }
+
+            using (var xmlOutput = File.Open(currentPackage.Path.FullName, FileMode.Truncate))
+            {
+                UpdateNuspec(xmlOutput, newVersion, currentPackage, xml);
+            }
+
+            return Task.CompletedTask;
+        }
+
+        private void UpdateNuspec(Stream fileContents, NuGetVersion newVersion,
```

Comment:
Looks like this was a hangover from using the Nuspec command as a template. Should be something like `UpdateDirectoryBuildTargets` (or something)